src/kem/kyber768/constant_time_ops.rs

```diff
@@ -0,0 +1,41 @@
+use crate::kem::kyber768::{CIPHERTEXT_SIZE, SHARED_SECRET_SIZE};
+
+// TODO: Attribute this to RustCrypto
+#[inline]
+fn is_non_zero(value: u8) -> u8 {
+    const SHIFT_BITS: usize = core::mem::size_of::<u64>() - 1;
+    let value = value as u64;
```

Comment:
Can we use `u32` here? `u64` seems overkill and might not be great on 32-bit systems.

Reply:
Switched to a simpler expression for now, we can complicate it depending on what LLVM outputs.